pr-data.csv

```diff
@@ -3472,9 +3472,9 @@ https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810ae
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_methodAnnotation_shouldNotCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldNotCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a
-https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID,,,
+https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID,Opened,https://github.com/AxonFramework/AxonFramework/pull/3142,
```

Comment:
3142 has been accepted. Did you run NonDex on the entire project to see if it finds some new tests?